apps/range-voting/app/components/Panels/VotePanelContent.js

```diff
@@ -152,8 +152,8 @@ class VotePanelContent extends React.Component {
     } = vote.data
 
     // TODO: Show decimals for vote participation only when needed
-    const voterParticipation = (participationPct * 10e15).toFixed(2)
-
+    const disParticipationPct = (participationPct).toFixed(2)
```

Comment:
disParticipation and disMinParticipation variable names seem not really helpful to understand their meaning, could we use more representative names for these variables?

Reply:
It's supposed to be short for 'display' but I can write the full thing out?